crates/protogen/proto/rpcsrv/common.proto

```diff
@@ -0,0 +1,31 @@
+// Types shared between our "external" and "internal" apis.
+//
+// See
+// <https://www.notion.so/glaredb/Distributed-Exec-f1e40e1dac174762b4f0fe4f6ac03ac9?pvs=4#c986d64150ac4b079d431d3d36f25f0f>
```

Comment:
Typically yes, but there's no other context around why this is split into its own file, so figured to add it in for quick reference. Once we have the proper internal/external definitions, we can remove this since the context will be clearer.